manifest.json

```diff
@@ -3,17 +3,24 @@
     "id": "peertube-search-index",
     "packaging_format": 1,
     "description": {
-        "en": "A search engine for PeerTube videos and channels",
-        "fr": "Un moteur de recherche de vidéos et chaînes PeerTube"
+        "en": "Search engine for PeerTube videos and channels",
+        "fr": "Moteur de recherche de vidéos et chaînes PeerTube"
     },
-    "version": "0.0.1-2021-02-17~ynh2",
+    "version": "2021.02.17~ynh3",
```

Comment:
Upstream version is still `0.0.1` in `package.json`, I would keep it that way.  Or is there an issue with the current versioning scheme?

Reply:
We just need to get rid of `-` for the CI to be happy...  so it will translate to `0.0.1.2021.02.17~ynh2` which is odd. We should choose either `0.0.1~ynh2` or `2021.02.17~ynh2`

Reply:
It's odd indeed, but I prefer to have a weird version number than preventing future upgrades if upstream decides to bump up from `0.0.1` 😅

Reply:
well I am also happy with this format `0.0.1.2021.02.17~ynh2`